test/test_nn.py

```diff
@@ -7096,6 +7095,43 @@ def test_fuse_module_eval_numerics(self, X, running_mean, running_var):
 
         self.assertEqual(Y_ref, Y_hat, msg="Conv+BN(non-affine) fusion results are off")
 
+    @set_default_dtype(torch.double)
+    @given(X=hu.tensor(shapes=((5, 3),), dtype=np.double),
+           running_mean=hu.tensor(shapes=(6,), dtype=np.double),
+           running_var=hu.tensor(shapes=(6,), dtype=np.double))
+    def test_fuse_module_linear_eval_numerics(self, X, running_mean, running_var):
+        inputs, _ = X
+
+        iC, oC = inputs.shape[1], len(running_mean[0])
+        inputs = torch.from_numpy(inputs)
+
+        linear = torch.nn.Linear(iC, oC, bias=True)
+        bn_ref = torch.nn.BatchNorm1d(oC)
+        bn_ref.running_mean = torch.from_numpy(running_mean[0])
+        bn_ref.running_var = torch.from_numpy(running_var[0])
```

Comment:
I copied the test from conv-bn fusion. I think it will be good to initialize BN weight and bias with random values also?